### ISSUE TITLE
Adding indicators from the add action page

### ISSUE THREAD

**Issue by ashbowe** [nmrf/sadata#29]
When I'm adding a new action and on that page click on 'add new indicator' to link it to I am only given a box to create an indicator without all of the required fields such assigned user and report due date. Is this intended?

**Comment by tmfrnz** [comment]
Yes, at least for now the "create new from multiselect" feature (opening in a "modal" that overlays the previous view) only allows creating new entities and their own attributes, but not any associations to other entities (incl connections and users). Currently associations have to be made from the principal entity edit screen.

Remember that originally this feature was only required for new category creation and it was only made available for the other entities as a "bonus" as it was possible with little extra effort. 

Allowing associations would require further analysis and some additional programming effort. Having the full "new entity form" in the "new entity modal" with all the options and associations could be problematic as
- we would not want the user to open another "new entity modal" from with the "new entity modal" and so on (which could be easily addressed by not giving that option)
- it may be beneficial for the user to focus only on the original entity and its associations (incl the newly created ones via the "new entity modal") and not at the same time also worry about other associations of the newly created entity

However I can see that the user assignment and due date creation for a newly created indicator may be a very useful exception but we may have to leave that for a future iteration

**Comment by tmfrnz** [comment]
have created issue in root repository: https://github.com/impactoss/impactoss-client/issues/388
